tests/test_ble_connection_discovery_client_targets.py

```
@@ -66,6 +74,39 @@ def test_connection_validator_client_is_connected_member_paths() -> None:
+    """Connection helper utilities should handle import fallback and inline cleanup execution."""
+    real_import = builtins.__import__
+
+    def _import_with_mock_failure(
+        name: str,
+        globals_arg: dict[str, object] | None = None,
+        locals_arg: dict[str, object] | None = None,
+        fromlist: tuple[str, ...] = (),
+        level: int = 0,
+    ) -> object:
+        if name == "unittest.mock":
+            raise ImportError("mock import unavailable")
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)